Assignment-1-6/form.php

```diff
@@ -0,0 +1,84 @@
+<?php
+    include 'emailValidate.php';
+
+    $firstName = $lastName = $marksStr = "";
+
+    $target_file = "";
```

Comment:
Let's use either `snake_case` or `camelCase` variable naming convention.
At line 4 we are using camelcase and at line 6 we are using snakecase, so let's not mix up.

Reply:
solved